### DISCUSSION TITLE
I would like to propose three new features

### DISCUSSION THREAD

**Discussion by ghost** [OneKeyHQ/app-monorepo#4401] [Ideas]
**Dear OneKey Team,**

As an active user of your desktop cryptocurrency wallet, I would like to propose three new features that, in my opinion, would significantly improve the user experience and save time:

**1. Application Configuration Export/Import:**
It would be very convenient to have the ability to export the entire application configuration, including address lists, settings, and other parameters, to an encrypted file. This would allow for easy transfer of the configuration to other devices without losing precious data and avoiding the need to re-enter all addresses and settings.

**2. Automatic Generation of the "All Networks" Page:**
Currently, enabling the display of all supported cryptocurrency networks requires manually enabling each network individually, which can be time-consuming. Automatically generating this page with the ability to select the desired networks would greatly simplify this process.

**3. Automatic Addition of All Tokens from an Address:**
It would be very convenient if the cryptocurrency wallet automatically added all tokens present at an address, even if they are spam tokens. Currently, each token needs to be added manually, which is extremely inconvenient. Automating this process would save a lot of time.

I am confident that implementing these features would make your cryptocurrency wallet even more user-friendly and attractive to users. I would be happy to discuss these proposals in more detail.

Thank you in advance for considering my ideas.

Best regards.

**Comment by revan-zhang** [comment]
Hi @manguro-code , currently our new v5.x version has been released. The all networks feature and the token auto detect feature has been redesigned. Let’s see if it meets expectations. We also have plans for the remaining issues.
